.github/actions/ubuntu16-build-action/build.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 make -j $(nproc) all
-make -j $(nproc) run_tests
+make -j run_tests
```

Comment:
Wow, I never knew about `-j` without any flags :brain:

Reply:
Makes two of us! That was a typo, but apparently `-j` without flags is a thing!

Reply:
Oh I thought it was intentional for maximum test pedal to the metal :laughing: